src/views/InvolvementsAll/index.jsx

```diff
@@ -173,86 +169,104 @@ const InvolvementsAll = () => {
                   </Select>
                 </FormControl>
               </Grid>
-              <Grid item xs={12} md={6} lg={3}>
-                <FormControl fullWidth variant="filled">
-                  <InputLabel id="activity-type">Type</InputLabel>
-                  <Select
-                    labelId="activity-type"
-                    id="activity-type"
-                    value={type}
-                    onChange={(event) => setType(event.target.value)}
-                  >
-                    <MenuItem label="All" value="">
-                      <em>All</em>
-                    </MenuItem>
-                    {types.map((type) => (
-                      <MenuItem value={type} key={type}>
-                        {type}
-                      </MenuItem>
-                    ))}
-                  </Select>
-                </FormControl>
-              </Grid>
             </Grid>
-          </CardContent>
-        </Card>
-      </Grid>
-
-      {!isOnline ? null : loadingProfile ? (
-        <GordonLoader />
-      ) : (
-        profile && <Requests profile={profile} session={selectedSession} />
-      )}
-
-      {loadingProfile ? (
-        <GordonLoader />
-      ) : (
-        profile && (
-          <Grid item xs={12} lg={8}>
-            <Card>
-              <CardHeader
-                title={`My ${myInvolvementsHeadingText} Involvements`}
-                style={{
-                  backgroundColor: gordonColors.primary.blue,
-                  color: gordonColors.neutral.grayShades[50],
-                }}
-              />
-              <CardContent>
-                {loading ? (
+            <TabContext value={tabIndex}>
+              <Grid item xs={16} marginTop={2}>
+                <TabList
+                  onChange={handleChangeTab}
+                  aria-label="involvements tabs"
+                  variant="fullWidth"
+                  centered
+                  indicatorColor="secondary"
+                >
+                  <Tab label="Personal" className={styles.tab} value="1" />
+                  <Tab label="Requests" className={styles.tab} value="2" />
+                  <Tab label="All" className={styles.tab} value="3" />
+                </TabList>
+              </Grid>
+              <TabPanel value="1">
+                {loadingProfile ? (
                   <GordonLoader />
                 ) : (
-                  <InvolvementsGrid
-                    involvements={myInvolvements}
-                    sessionCode={selectedSession}
-                    noInvolvementsText={myInvolvementsNoneText}
-                  />
+                  profile && (
+                    <Grid item xs={12} lg={12}>
+                      {loading ? (
+                        <GordonLoader />
+                      ) : (
```

Comment:
You've nested two GordonLoaders inside of conditional logic on this page. It can look kind of glitchy while loading when you do that. I would combine them so that line 188 looks like:
`{loadingProfile || loading ? (`
Then you can remove the logic in lines 193-195